templates/hotgraphic.hbs

```diff
@@ -17,7 +17,13 @@
         {{/if}}
         {{#each _items}}
           <button class="base hotgraphic-graphic-pin component-item-text-color {{#if _isVisited}}visited{{/if}}" data-index="{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" aria-label="{{title}}.{{#if _isVisited}} {{../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
-            <div class="hotgraphic-graphic-pin-icon component-item-color icon icon-pin item-{{@index}}"></div>
+            {{#if this._pin.src}}
```

Comment:
Minor: I think `this` can be omitted as the context in `#each` will always be the item `{{#if _pin.src}}` - just makes things easier to read

Reply:
Thanks :)